src/services/filter.ts

```diff
@@ -66,18 +48,36 @@ export async function filterFeatureByPoiAndLine(
           line as LineNumber
         )
     )
-    .map((f) => f.getId()!)
-  layer.removeFeaturesById(featuresToDelete)
+    .map((f) => f.getId()!.toString())
+  layer.featureVisibility.hideObjects(featuresToHide)
 }
 
 export async function filterFeatureByPoiAndStation(
   rennesApp: RennesApp,
   station: string
 ) {
   const layer = await rennesApp.getLayerByKey(RENNES_LAYER.poi)
-  const featuresToDelete = layer
+  layer.setGlobalHider(new GlobalHider())
+  const featuresToHide = layer
     .getFeatures()
     .filter((f) => f.getProperties()['station_nom'] !== station)
     .map((f) => f.getId()!)
-  layer.removeFeaturesById(featuresToDelete)
+  layer.featureVisibility.hideObjects(featuresToHide)
+}
+
+async function filterFeatureByLayerAndKeyAndValue(
+  rennesApp: RennesApp,
+  layerName: string,
+  featureKey: string,
+  featureValue: string
+) {
+  const layer = await rennesApp.getLayerByKey(layerName)
+  layer.setGlobalHider(new GlobalHider())
```

Comment:
what is this `GlobalHider` used for? I tried to read the doc, but I am not sure I get the explanation. It may be useful for styling